.github/workflows/run-match.yml

```diff
@@ -0,0 +1,75 @@
+name: Simulator match
+
+on:
+  push: {}
+  release:
+    types: [ created ]
+
+jobs:
+  run-comp-match:
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        shell: bash
+        working-directory: ${{ github.workspace }}
```

Comment:
Is this stuff necessary?

Reply:
Only when using docker otherwise `/bin/sh` is used